src/python/zquantum/core/hamiltonian_test.py

```diff
@@ -54,3 +166,115 @@ def test_group_comeasureable_terms_greedy_sorted(
 ):
     groups = group_comeasureable_terms_greedy(qubit_operator, sort_terms=sort_terms)
     assert groups == expected_groups
+
+@pytest.mark.parametrize(
+    "interactionrdm, qubitoperator, sort_terms",
+    [
+        (
+            rdms, h2_hamiltonian, False,
+        ),
+        (
+            rdms, h2_hamiltonian, True,
+        ),
+
+    ],
+)
+
+def test_get_expectation_values_from_rdms(interactionrdm,
+    qubitoperator, sort_terms
+):
+    expecval = get_expectation_values_from_rdms(interactionrdm, qubitoperator, sort_terms)
+    assert len(expecval.values) == len(qubitoperator.terms) - 1
+    if not sort_terms:
+        coeff = np.array(list(qubitoperator.terms.values()))
+        coeff = coeff[1:]
+    else:
+        sorted_qubitoperator = QubitOperator((), 0.0)
+        terms_iterator = sorted(
+            qubitoperator.terms.items(), key=lambda x: abs(x[1]), reverse=True
+        )
+        coeff = []
+        for term, coefficient in terms_iterator:
+            if not term == ():
```

Comment:
It is more Pythonic to use `!=` operator instead of combination of `not` and `==`, i.e. `if term != ()`.

Reply:
Fixed!